src/Fixer/Import/NoUnusedImportsFixer.php

```diff
@@ -114,9 +114,18 @@ function (NamespaceUseAnalysis $useDeclaration) use ($namespace) {
         }
     }
 
+    /**
+     * @param Tokens            $tokens
+     * @param NamespaceAnalysis $namespace
+     * @param array<int, int>   $ignoredIndexes
+     * @param string            $shortName
+     *
+     * @return bool
+     */
     private function importIsUsed(Tokens $tokens, NamespaceAnalysis $namespace, array $ignoredIndexes, $shortName)
```

Comment:
Can we rename this method to `isImportUsed`?

Reply:
I prefer the current name, it feels more natural.

Reply:
How come it is more natural? The question "is import used?" is the right way to ask, not "import is used?". Also `is` at the beginning is the "classic" prefix for returning boolean.

Reply:
I always try to think of methods as orders to give rather than questions to ask: _"Tell me whether this **import is used**"_ vs. _"**Is** this **import used**?"_.

Reply:
> I always try to think of methods as orders

Me too, when possible, yet this case is yes/no question after all.

Reply:
Method renamed.